internal/client/client.go

```diff
@@ -775,6 +775,39 @@ func (a *AuthApiClient) ListTasks() ybmclient.ApiListTasksRequest {
 	return a.ApiClient.TaskApi.ListTasks(a.ctx, a.AccountID)
 }
 
+func (a *AuthApiClient) ListRbacRoles() ybmclient.ApiListRbacRolesRequest {
```

Comment:
nit: this function is repeated in another PR, please take care of it while merging.